core/math/math_funcs.h

```diff
@@ -411,7 +410,7 @@ class Math {
 			uint64_t i;
 		} u;
 		u.d = g;
-		u.i &= (uint64_t)9223372036854775807ll;
+		u.i &= (uint64_t)9'223'372'036'854'775'807ll;
```

Comment:
Could maybe be replaced by a constexpr to calculate it at compile time instead of hardcoding.

Reply:
Or, `~(1ll << 63)`, or `7F'FF'FF'FF'FF'FF'FF'FF`